CHANGELOG.unreleased.md

```diff
@@ -44,6 +44,50 @@ const link = <a href="example.com">http://example.com</a>;
 
 -->
 
+#### Glimmer: Fix handling of whitespace and line breaks ([#6354] by [@chadian])
+
+This fixes a variety of whitespace and line break usecases within Glimmer.
+
+<!-- prettier-ignore -->
+```hbs
+{{! Input }}
+<SomeComponent />{{name}}
+
+Some sentence with  {{dynamic}}  expressions.
+
+
+
+sometimes{{nogaps}}areimportant<Hello></Hello>
+{{name}}  is your name
+
+{{! Output (Prettier stable) }}
+<SomeComponent />
+{{name}}
+Some sentence with
+{{dynamic}}
+expressions.
+
+
+
+sometimes
+{{nogaps}}
+areimportant
+<Hello />
+{{name}}
+is your name
+
+{{! Output (Prettier master) }}
+<SomeComponent />{{name}}
+
+Some sentence with {{dynamic}} expressions.
+
+
+
+sometimes{{nogaps}}areimportant
+<Hello />
+{{name}} is your name
+```
```

Comment:
Please fix format, look on other items in chanelog

Reply:
I've updated it basing it off of the other Handlebars entry in the changelog. Is it correct now?